lib/bio_monitor/routine_monitor.ex

```diff
@@ -158,6 +224,41 @@ defmodule BioMonitor.RoutineMonitor do
     end
   end
 
+  defp is_offset_stable?(ph_cal) do
+    case Enum.count(ph_cal.values) >= 10 do
+      true ->
+        case Math.Enum.mean(ph_cal.values) do
+          nil -> false
+          mean ->
+            oscillation = mean - List.last(ph_cal.values)
+            oscillation <= @ph_oscillation_tolerance && oscillation >= -@ph_oscillation_tolerance
```

Comment:
`Kernel.abs(oscillation) <= @ph_oscillation_tolerance`?

Reply:
I like it 👌